tornado/netutil.py

```diff
@@ -110,8 +111,26 @@
 _DEFAULT_BACKLOG = 128
 
 
+# This workaround is needed for Python <= 2.7.5,
```

Comment:
I don't think we should have this workaround. It's fine if this new feature is only available to people who have a recent enough version of Python. (especially since all versions of python older than 2.7.9 have a very outdated `ssl` module; any extra incentive to get people to upgrade is a good thing)